src/App.js

```diff
@@ -1,43 +1,82 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import Joke from "./components/Joke";
 import PageHeader from "./components/PageHeader";
+import SearchInput from "./components/SearchInput";
+import Footer from './components/Footer'
 
-class App extends React.Component {
-  constructor() {
-    super()
-    this.state = {
-      jokeData: {
-        results: []
-      }
-    }
+const headers = { accept: "application/json" }
+let url;
+
+const LOADING_STATE = {
+  resting: "resting",
+  loaded: "Loaded",
+  loading: "Loading... please wait",
+  error: "Something went worng..."
+};
+
+function App(props) {
+  const [jokeData, setJokeData] = useState({ })
+  const [searchTerm, setSearchTerm] = useState("hat")
+  const [loadingState, setLoadingState] = useState(LOADING_STATE.resting)
+  
+
+  function handleChange(event) {
+    const {value} = event.target
+    setSearchTerm(value);
+    console.log("hello")
+  };
```

Comment:
@arb This handlechange is no longer working. Well it's not console logging anything when typing in the field.